doc/dev/msgr2.rst

```diff
@@ -556,7 +556,7 @@ Once the handshake is completed, both peers have setup their compression handler
     uint32_t  method
 
   - the server determines whether compression is possible according to its' configuration.
-  - if it is possible, it will pick its' most prioritizied compression method that is also supprorted by the client.
+  - if it is possible, it will pick its' most prioritized compression method that is also supported by the client.
```

Comment:
Since we're fixing this line, s/its'/the/

Should be "its" but I think "the" flows better.